pool/client.go

```diff
@@ -864,10 +860,18 @@ func (c *Client) process() {
 
 				case Submit:
 					err := c.handleSubmitWorkRequest(c.ctx, req, allowed)
+					if errors.Is(err, errs.NetworkDifficulty) {
+						// Submissions less than the network difficulty should
+						// not be treated as errors.
+						log.Trace(err)
```

Comment:
For the reasons above - I think debug or info are fine here. This is not an excessive amount of info, or logged excessively often